doc/kios.md

```diff
@@ -1 +1,726 @@
+}
+```
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id": 1,
+        "kios_name": "kalimantan",
+        "address": "bekasi timur raya no. 159 A",
+        "branches": [
+            {
+                "branch_code": "02629"
+            }
+        ]
+    }
+}
+```
+
+#### Case : Positive Case 2
+
+Request Payload :
+```
+{
+    "kios_name": "kalimantan",
+    "address": "jl. k. h. moch mansyur no. 32 jembatan 5",
+    "branch_code": "02629"
+}
+```
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id": 1,
+        "kios_name": "kalimantan",
+        "address": "jl. k. h. moch mansyur no. 32 jembatan 5",
+        "branches": [
+            {
+                "branch_code": "02629"
+            }
+        ]
+    }
+}
+```
+
+## <a name="view"></a>View
+
+### Endpoint
+GET /kios/:id
+
+### Header
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+Email | myemail@gmail.com
+
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+400 | Bad Request | Bad request payload  
+404 | Not Found | Kios not found in database  
+403 | Forbidden | kios_id not owned by user login 
+500 | Internal Server Error | some un-handle error in server 
+200 | OK | OK
+```
+{
+    "status_code": "CDC-400",
+    "status_message": "Bad Request",
+    "data": null
+}
+```
+
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id": 1,
+        "kios_name": "mandala",
+        "address": jl. "bekasi timur raya no. 159 A",
+        "branches": [
+            {
+                "branch_code": "02629"
+            }
+        ]
+    }
+}
+```
+
+### Logic
+
+#### Validation
+- kios_id : is owned by user login
+
+### Scenario Test
+
+#### Case : Negative Case
+
+response HTTP Status Code : 403
+
+Response Payload : 
+```
+{
+    "status_code": "cdc-403",
+    "status_message": "kios_id not owned by user login",
+    "data": null
+}
+```
+
+#### Case : Positive Case
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id": 1,
+        "kios_name": "kalimantan",
+        "address": "jl. k. h. moch mansyur no. 32 jembatan 5",
+        "branches": [
+            {
+                "branch_code": "06502"
+            }
+        ]
+    }
+}
+```
+
+## <a name="delete"></a>Delete
+
+### Endpoint
+DELETE /kios/:id
+
+### Header
+Key | Value 
+--- | ---
+Content-Type | *
+Accept | application/json
+Email | myemail@gmail.com
+
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+404 | Not Found | Kios not found in database  
+403 | Forbidden | id not owned by user login
+500 | Internal Server Error | some un-handle error in server
+204 | No Content | Delete data kios
+
+```
+{
```

Comment:
204 no content. it's mean really no content, so you shouldn't have response. delete all response in body if the response status code is 204.